src/penn_chime/models.py

```diff
@@ -44,6 +44,14 @@ def __init__(self, p: Parameters) -> SimSirModel:
             p.known_infected / infected if infected > 1.0e-7 else None
         )
 
+
+        #self.hospitalized_t = p.current_hospitalized
+        #self.n_days_since_first_hospitalized = p.n_days_since_first_hospitalized
+        if p.n_days_since_first_hospitalized is not None:
+            # do not shorten this to `if foo:` form. if you do and n_days_since_first_hospitalized is 0,
+            # program will crash.
+            p.doubling_time = self.argmin_dt(np.linspace(1, 15, 29), p)
```

Comment:
Would it make sense to fit to a finer precision than seemingly half days? I feel like this could introduce some wild discretization error in the forecasts. If there's a performance concern in increasing the precision for the whole range, you can iteratively refine the fit, by doing successive `argmin_dt` calls in shrinking neighborhoods around the previous value found.

Reply:
agreed, but not in the first iteration